### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by GeertvanHorrik**
This PR contains several improvements. I will keep pushing changes to this PR.

### CODE REVIEW

File: src/GitReleaseNotes/GitReleaseNotesEnvironment.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿namespace GitReleaseNotes
+{
+    public static class GitReleaseNotesEnvironment
+    {
+        #region Fields
```
Comment (JakeGinnivan) [inline src/GitReleaseNotes/GitReleaseNotesEnvironment.cs]: Death to regions! :P

Comment (GeertvanHorrik) [inline src/GitReleaseNotes/GitReleaseNotesEnvironment.cs]: Oops, resharper setting. Will remove them

Comment (GeertvanHorrik) [inline src/GitReleaseNotes/GitReleaseNotesEnvironment.cs]: btw, just install this: https://visualstudiogallery.msdn.microsoft.com/0ca60d35-1e02-43b7-bf59-ac7deb9afbca

**Comment by JakeGinnivan** [comment]
Looks grand. Few things i'd change but not too fussed as there is so much wrong with this codebase and this makes great steps.

**Comment by JakeGinnivan** [comment]
Let me know when you want it merged

**Comment by JakeGinnivan** [comment]
I will also just merge before I want to do any more work on this. Hopefully next few days

**Comment by GeertvanHorrik** [comment]
I have no idea how you calculate the releases. Was checking out the tags, but somehow it doesn't work really good. I hoped to find you on skype (no gitter for this repository).

**Comment by GeertvanHorrik** [comment]
I think it's ready for the first merge. It now works in JIRA

**Pull request merged**